tests/app/main/views/test_add_service.py

```diff
@@ -395,12 +404,36 @@ def test_email_auth_user_creates_service_with_email_auth_permission(
     assert "email_auth" in mock_update_service.call_args[1]["permissions"]
 
 
+@pytest.mark.parametrize(
+    "organisation",
+    (
+        None,
+        organisation_json(ORGANISATION_ID),
+    ),
+)
+def test_join_or_add_service_page_redirects_without_permission(
```

Comment:
this test doesn't actually test that the we redirect, but that we throw 403, so maybe name could reflect that? 

```suggestion
def test_join_or_add_service_page_403s_without_permission(
```